pkg/api/http_server.go

```diff
@@ -255,7 +257,7 @@ func ProvideHTTPServer(opts ServerOptions, cfg *setting.Cfg, routeRegister routi
 	accesscontrolService accesscontrol.Service, dashboardThumbsService thumbs.DashboardThumbService, navTreeService navtree.Service,
 	annotationRepo annotations.Repository, tagService tag.Service, searchv2HTTPService searchV2.SearchHTTPService,
 	queryLibraryHTTPService querylibrary.HTTPService, queryLibraryService querylibrary.Service, oauthTokenService oauthtoken.OAuthTokenService,
-	statsService stats.Service, authnService authn.Service,
+	statsService stats.Service, authnService authn.Service, dataSourceCacheCfgService querycaching.DatasourceCacheConfig,
```

Comment:
I dont think the oss grafana server shouldn't depend on an enterprise service. Is this temporary?

Reply:
It's not relying on an enterprise service per se, it just needs an implementation of the interface, and that impl is OSS or Enterprise depending on the build. I believe this is the Grafana convention for this sort of thing.

Reply:
cool, that makes sense 👍